spec/API_specification/indexing.md

```diff
@@ -105,11 +105,6 @@ Slice syntax must have the following defaults. Let `n` be the axis (dimension) s
 -   If `k` is greater than `0` and `j` is not provided (e.g., `0::2`), `j` must equal `n`.
 -   If `k` is less than `0` and `i` is not provided (e.g., `:10:-2`), `i` must equal `n-1`.
 -   If `k` is less than `0` and `j` is not provided (e.g., `0::-2`), `j` must equal `-n-1`.
-
-Using a slice to index a single array axis must adhere to the following rules. Let `n` be the axis (dimension) size.
```

Comment:
This line should not be removed. Otherwise, the item defined by L113 will look like it belongs to the list ending at L107.

Reply:
Well that was also confusing to me. The two lists both seem like they are about the same thing (the behavior of a slice on an axis of size `n`).